client/src/pages/spectrogram/components/PluginsPane.tsx

```diff
@@ -158,7 +162,33 @@ export const PluginsPane = ({ cursorsEnabled, handleProcessTime, meta, setMeta }
           });
 
           setModalOpen(true);
+        } else {
+          // Data file
+          const samples_base64 = data.data_output[0]['samples'];
+          const samples = Buffer.from(samples_base64, 'base64');
+
+          const a = document.createElement('a');
+          document.body.appendChild(a);
+          a.style = 'display: none';
+          var blob = new Blob([samples], { type: data.data_output[0]['data_type'] }),
```

Comment:
You would usually use const or let in react:
https://sentry.io/answers/difference-between-let-and-var-in-javascript/#:~:text=The%20difference%20between%20let%20and,in%20which%20they're%20declared.

Reply:
done.